projects/1.16.1/assets/astral-sorcery/astralsorcery/lang/zh_cn.json

```
@@ -1,1152 +1,1153 @@
-
-  "perk.astralsorcery.special.focus.gelu.name": "定向：寒冰座",
-  "perk.astralsorcery.special.focus.gelu.desc.1": "每个已获得的技能额外增加+2%%的技能效率。",
-  "perk.astralsorcery.special.focus.gelu.desc.2": "不与其他的技能效率强化效果连用。",
-  "perk.astralsorcery.special.focus.gelu.desc.3": "",
-  "perk.astralsorcery.special.focus.gelu.desc.4": "你只能拥有一个固有技能。",
-  "perk.astralsorcery.special.focus.ulteria.name": "定向：避役座",
-  "perk.astralsorcery.special.focus.ulteria.desc.1": "每个未使用的技能点额外+5%%技能经验。",
-  "perk.astralsorcery.special.focus.ulteria.desc.2": "",
-  "perk.astralsorcery.special.focus.ulteria.desc.3": "你只能拥有一个固有技能。",
-  "perk.astralsorcery.special.focus.vorux.name": "定向：贪狼座",
-  "perk.astralsorcery.special.focus.vorux.desc.1": "每个已获得的技能额外增加+1%%的技能效率。",
-  "perk.astralsorcery.special.focus.vorux.desc.2": "你将无法获得技能经验。",
-  "perk.astralsorcery.special.focus.vorux.desc.3": "",
-  "perk.astralsorcery.special.focus.vorux.desc.4": "你只能拥有一个固有技能。",
-  "perk.astralsorcery.special.focus.alcara.name": "定向：振变座",
-  "perk.astralsorcery.special.focus.alcara.desc.1": "增加与减少技能经验",
-  "perk.astralsorcery.special.focus.alcara.desc.2": "适用与所有技能效率强化的替换",
-  "perk.astralsorcery.special.focus.alcara.desc.3": "",
-  "perk.astralsorcery.special.focus.alcara.desc.4": "你只能拥有一个固有技能。",
-
-  "perk.astralsorcery.special.connector.evorsio.name": "查克拉的无常性质",
-  "perk.astralsorcery.special.connector.discidia.name": "查克拉的破坏性质",
-  "perk.astralsorcery.special.connector.armara.name": "查克拉的连续性质",
-  "perk.astralsorcery.special.connector.vicio.name": "查克拉的变化性质",
-  "perk.astralsorcery.special.connector.aevitas.name": "查克拉的构造性质",
-  "perk.astralsorcery.special.connector.key.evorsio.name": "顿悟：无常之道",
-  "perk.astralsorcery.special.connector.key.evorsio.desc.1": "其他“顿悟”技能不受此影响",
-  "perk.astralsorcery.special.connector.key.evorsio.desc.2": "需要与现有的强化有所联系",
-  "perk.astralsorcery.special.connector.key.discidia.name": "顿悟：毁灭之道",
-  "perk.astralsorcery.special.connector.key.discidia.desc.1": "其他“顿悟”技能不受此影响",
-  "perk.astralsorcery.special.connector.key.discidia.desc.2": "需要与现有的强化有所联系",
-  "perk.astralsorcery.special.connector.key.armara.name": "顿悟：连续之道",
-  "perk.astralsorcery.special.connector.key.armara.desc.1": "其他“顿悟”技能不受此影响",
-  "perk.astralsorcery.special.connector.key.armara.desc.2": "需要与现有的强化有所联系",
-  "perk.astralsorcery.special.connector.key.vicio.name": "顿悟：变化之道",
-  "perk.astralsorcery.special.connector.key.vicio.desc.1": "其他“顿悟”技能不受此影响",
-  "perk.astralsorcery.special.connector.key.vicio.desc.2": "需要与现有的强化有所联系",
-  "perk.astralsorcery.special.connector.key.aevitas.name": "顿悟：构造之道",
-  "perk.astralsorcery.special.connector.key.aevitas.desc.1": "其他“顿悟”技能不受此影响",
-  "perk.astralsorcery.special.connector.key.aevitas.desc.2": "需要与现有的强化有所联系",
-
-  "perk.astralsorcery.special.arc_chain_1.name": "弧光链",
-  "perk.astralsorcery.special.arc_chain_1.desc": "额外增加1倍时间的环状闪电弧光链攻击",
-  "perk.astralsorcery.special.arc_chain_2.name": "弧光链",
-  "perk.astralsorcery.special.arc_chain_2.desc": "额外增加2倍时间的环状闪电弧光链攻击"
-}
+﻿{
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
﻿{
```

---

理论上来说可以修复一些小问题

---

但实际上不能